Git-commit: 43b33a48c03da14bc63d1a4f3c36a5cb88379c0d
From: Sang Yan <sangyan@huawei.com>
Date: Fri, 19 Feb 2021 14:28:10 +0800
Subject: [PATCH] arm64: Enable cpu park on openEuler
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.1.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

hulk inclusion
category: feature
bugzilla: 48159
CVE: N/A

------------------------------

Enable cpu park on openEuler by default.

Signed-off-by: Sang Yan <sangyan@huawei.com>
Reviewed-by: Xiongfeng Wang <wangxiongfeng2@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index c290ea0aaa85..89d83600c409 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -274,6 +274,7 @@ CONFIG_FIX_EARLYCON_MEM=y
 CONFIG_PGTABLE_LEVELS=3
 CONFIG_ARCH_SUPPORTS_UPROBES=y
 CONFIG_ARCH_PROC_KCORE_TEXT=y
+CONFIG_ARM64_CPU_PARK=y
 
 #
 # Platform selection
-- 
2.31.1

